app/models/Org.js

```diff
@@ -0,0 +1,35 @@
+import mongoose, { Schema } from 'mongoose';
+
+//do not delete this line or the Founder schema will not be registered properly
+import Founder from './Founder';
+
+const FounderSchema = require('mongoose').model('Founder').schema;
```

Comment:
We've already imported `mongoose` on line one, can we change this line to take advantage of that instead of importing again, but in ES5 sytax?